sunbird-rc-esignet-integration-impl/src/main/java/io/mosip/esignet/sunbirdrc/integration/service/SunbirdRCAuthenticationService.java

```diff
@@ -76,16 +75,11 @@ public void initialize() throws KycAuthException {
         validateProperty("mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.individual-id-field");
         validateProperty("mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.registry-search-url");
         validateProperty("mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.field-details");
```

Comment:
can we remove the above 3 validateProperty calls ? Since it will be taken care by @value itself. Also lets remove the env bean and this validateProperty method also.

Reply:
"On init of the plugin class validate if properties are configured correctly, the individual-id-field should be available as part of the field details" - this validation is missing as per the confluence page.

Reply:
added the validation